tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -398,3 +416,23 @@ Verify Anaconda Success Message Based On Version
     ELSE
         Success Message Should Contain    ${ANACONDA_DISPLAYED_NAME_LATEST}
     END
+
+Verify RHODS Notification After Logging Out
+    [Documentation]     Logs out from RHODS Dashboard and then relogin to check notifications
+    Go To    ${ODH_DASHBOARD_URL}
+    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    RHODS Notification Drawer Should Contain  message=Notebook image build TensorFlow failed
+
+Restart Failed Build and Close Browser
+    [Documentation]     Deletes failed build and starts new build , Closes All Browsers
+    [Arguments]     ${failed_build_name}  ${build_config}
+    Delete Failed Build And Start New One  namespace=redhat-ods-applications  failed_build_name=${failed_build_name}  build_config_name=${build_config}
+    Dashboard Test Teardown
+
+Verify Notifications After Build Is Complete
+    [Documentation]  Verifies Notifications after build status is complete
+    RHODS Notification Drawer Should Contain  message=2
+    RHODS Notification Drawer Should Contain  message=of
+    RHODS Notification Drawer Should Contain  message=3
+    RHODS Notification Drawer Should Contain  message=builds completed successfully
```

Comment:
why splitting on four lines? You coudl search for "2 of 3 builds completed successfully"

Reply:
Updated and tested on jenkins ; https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1012/